<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InserisciNuovoAudioDialog</class>
 <widget class="QDialog" name="InserisciNuovoAudioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibreriaDigitale</string>
  </property>
  <widget class="QLabel" name="autoreALabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Autore:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="autoreATextEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="audioInseritoLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>220</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;Il seguente audio Ã¨ stato correttamente aggiunto alla libreria:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="casaEditriceALabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Casa discografica:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="inserisciNuovoAudioButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border-radius: 8px;
background-color:#333333; 
border: none;
color: white;
padding: 2px;
text-align: center;
font-size: 15px;
font-family: &quot;Garamond&quot;;
}

:hover {
background-color: #676767;
}

:pressed{
border: 1px solid #E0E2E2;
}</string>
   </property>
   <property name="text">
    <string>Inserisci</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="durataATextEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="casaDiscATextEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="titoloATextEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 12pt;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLabel" name="dataALabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Durata:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="audioInseritoTextEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
border: 1px solid #333333;
font-family: &quot;Garamond&quot;;
border-radius: 4px;
font-size: 10pt;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titoloALabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
color: #fdfdff;
font-family: &quot;Garamond&quot;;
font-size: 13pt;
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#fdfdff;&quot;&gt;Titolo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
